CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/LabelDropper.java

```diff
@@ -133,6 +133,8 @@ public void edit(final GraphWriteMethods wg, final PluginInteraction interaction
 
             } catch (final UnsupportedFlavorException | IOException | ClassNotFoundException ex) {
                 Exceptions.printStackTrace(ex);
+            } catch (final ClassCastException ex) {
+                //Do nothing
```

Comment:
@antares1470 would you be able to explain a bit more of some of the scenarios that lead to this happening "e.g. dropping a attribute label causes this"; something like that so we know why we are doing nothing? Thank you.

Reply:
Done